Marlin/pins_RAMPS.h

```diff
@@ -281,9 +314,15 @@
 
       #define BEEPER_PIN        37
 
+#if ENABLED(CR10_stockdisplay)
+      #define BTN_EN1           17 //31
+      #define BTN_EN2           23 //33
+      #define BTN_ENC           35 //35
+#else
       #define BTN_EN1           31
       #define BTN_EN2           33
       #define BTN_ENC           35
+#endif
```

Comment:
Indentation counts. Please strive to emulate the indentation style used in the rest of the codebase.